src/cmd/run.js

```diff
@@ -322,6 +322,8 @@ export class ExtensionRunner {
 
   run(profile: FirefoxProfile): Promise<FirefoxProcess> {
     const {firefoxApp, firefox} = this;
-    return firefoxApp.run(profile, {firefoxBinary: firefox});
+    return firefoxApp.run(
+        profile, {firefoxBinary: firefox, binaryArgs: ['-jsconsole']}
```

Comment:
Do I need to pass only the `jsconsole: true` parameter into `ExtensionRunner.run()`, like this
`run(profile: FirefoxProfile, jsconsole: true): Promise<FirefoxProcess> {`